Build failures when running electron-rebuild

I'm getting the following build errors when I attempt to run `electron-rebuild` with grpc.

```
../deps/grpc/third_party/boringssl/crypto/curve25519/spake25519.c:26:10: fatal error: '../../third_party/fiat/internal.h' file not found
#include "../../third_party/fiat/internal.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Here are the versions I'm using:

```
  "dependencies": {
    "grpc": "^1.11.0"
  },
  "devDependencies": {
    "electron": "^1.8.6",
    "electron-rebuild": "^1.7.3"
  }
```

--- a/packages/grpc-native-core/package.json
+++ b/packages/grpc-native-core/package.json
@@ -71,6 +71,7 @@
     "deps/grpc/third_party/boringssl/crypto/**/*.{c,cc,h}",
     "deps/grpc/third_party/boringssl/include/**/*.{c,cc,h}",
     "deps/grpc/third_party/boringssl/ssl/**/*.{c,cc,h}",
+    "deps/grpc/third_party/boringssl/third_party/**/*.{c,h}",
     "deps/grpc/third_party/abseil-cpp/absl/**/*.{h,hh}",
     "binding.gyp"
   ],

--- a/packages/grpc-native-core/templates/package.json.template
+++ b/packages/grpc-native-core/templates/package.json.template
@@ -73,6 +73,7 @@
       "deps/grpc/third_party/boringssl/crypto/**/*.{c,cc,h}",
       "deps/grpc/third_party/boringssl/include/**/*.{c,cc,h}",
       "deps/grpc/third_party/boringssl/ssl/**/*.{c,cc,h}",
+      "deps/grpc/third_party/boringssl/third_party/**/*.{c,h}",
       "deps/grpc/third_party/abseil-cpp/absl/**/*.{h,hh}",
       "binding.gyp"
     ],

